packages/node_modules/@ciscospark/react-component-utils/src/activity.js

```diff
@@ -5,7 +5,7 @@ import {isImage} from './files';
  * Constructs a default activity
  *
  * @export
- * @param {any} conversation
+ * @param {object} conversation
```

Comment:
We're not using it that way, but we are doing `id: conversation.id || conversation,` below. We need the full conversation object for retries now, so I'll remove the ability to just pass an ID.